src/cli/cli.cpp

```diff
@@ -2932,17 +2843,33 @@ void Interpreter::ProcessJoinerPort(int argc, char *argv[])
 }
 #endif
 
+void Interpreter::ProcessBlacklist(int argc, char *argv[])
+{
+    ProcessAddressFilter(OT_MAC_ADDRESSFILTER_MODE_BLACKLIST, argc, argv);
```

Comment:
Would it be simpler to adjust the CLI so that it mirrors the API updates more closely?